assets/css/v2/lcd_common/departures/later_departures.scss

```diff
@@ -0,0 +1,83 @@
+@use "sass:math";
+
+.later-departures {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  min-height: 280px;
```

Comment:
I saw the commit message about "departures with additional information" but wasn't sure what that meant; why is the explicit height needed?

Reply:
I was referring to stuff like this:

<img width="591" alt="image" src="https://github.com/mbta/screens/assets/1699281/33ac08d5-bddf-4741-93b8-57a7ed46068e">

Without the `min-height` these caused layout shift

Reply:
Ah, now I see (literally) what you mean. It would be nice if we didn't have to hard-code this... total armchair-engineering here, what if all "later departures" were rendered at all times, horizontally next to each other but all taking up 100% width, so only one is visible at a time? Then "rotating" the array by `currentDepartureIdx` places the current departure in the first position, making it visible — but the heights of all departures are always contributing to the height of the container.

```
visible | hidden in overflow
A       | B C D E
[rotate]
B       | C D E A
```

Anyway I do think the current solution is fine, but if we stick with that, maybe we should add a comment explaining why it's here.

Reply:
Oh I love this idea! Very clever. I'll give it a shot.

Reply:
Huh, I was thinking of just the departure rows themselves being rendered all next to each other, rather than a full copy of Later Departures for each rotation. This works too though!